webviz_config/static/assets/webviz_config.css

```diff
@@ -22,12 +22,15 @@
     margin: auto;
     max-width: 90%;
     max-height: 90vw;
+    margin-top: 16px;
+    margin-bottom: 16px;
 
 }
 
 ._markdown_image_caption {
 
     display: block;
     text-align: center;
-
+    margin-top: 16px;
+    margin-bottom: 16px;
```

Comment:
Do you have a screenshot of how this looks readily available? 🙂 Should we let `margin-top < margin-bottom` for the image caption, i.e. that the image caption text is closer to the image it belongs to compared with next element below, or is this already the case?